dotCMS/src/main/java/com/dotmarketing/util/starter/ImportStarterUtil.java

```diff
@@ -0,0 +1,1079 @@
+                    }
+                } else {
+                    if (obj instanceof Tree) {
+                        final Tree t = (Tree) obj;
+
+
+                        DotConnect dc = new DotConnect();
+                        List<String> inodeList = new ArrayList<>();
+                        dc.setSQL("select inode from inode where inode = ? or inode = ?");
+                        dc.addParam(t.getParent());
+                        dc.addParam(t.getChild());
+                        inodeList.addAll(dc.loadResults());
+                        dc.setSQL("select id from identifier where id = ? or id = ?");
+                        dc.addParam(t.getParent());
+                        dc.addParam(t.getChild());
+                        inodeList.addAll(dc.loadResults());
+                        if (inodeList.size() > 0) {
+                            TreeFactory.saveTree(t);
+                        } else {
+                            Logger.warn(this.getClass(), "Skipping tree- no matching inodes: {parent=" + t.getParent()
+                                    + ", child=" + t.getChild() + "}");
+                        }
+
+                    } else if (obj instanceof MultiTree) {
+                        final MultiTree t = (MultiTree) obj;
+                        APILocator.getMultiTreeAPI().saveMultiTree(t);
+
+                    } else {
+
+                        Logger.debug(this, "Saving the object: " + obj.getClass() + ", with the values: " + obj);
+
+
+                        HibernateUtil.save(obj);
+                        HibernateUtil.flush();
+
+                    }
+                }
+
+                HibernateUtil.getSession().flush();
+
+
+            }
+            if (identityOn) {
+                turnIdentityOffMSSQL(tableName);
+            }
+        }
+
+    }
+
+    /**
+     * Populates a list of {@link Contentlet} given a JSON file
+     * @param file
+     * @return {@link List<Contentlet}
+     * @throws IOException
+     */
+    private List<Contentlet> getContentletList(final File file) throws IOException {
+        return ContentletJsonHelper.INSTANCE.get().readContentletListFromJsonFile(file).stream().map(
+                cont -> {
+                    try {
+                        return APILocator.getContentletJsonAPI().toMutableContentlet(cont);
+                    } catch (DotDataException | DotSecurityException e) {
+                        e.printStackTrace();
+                        return new Contentlet();
+                    }
+                }).collect(Collectors.toList());
+    }
+
+    private User loadUserFromIdOrEmail(final User u) throws DotDataException, DotSecurityException {
+        User user = null;
+
+        try {
+            user = APILocator.getUserAPI().loadUserById(u.getUserId(), APILocator.getUserAPI().getSystemUser(), false);
+        } catch (Exception e1) {
+            Logger.info(this, "couldn't find user by ID going to lookup by email address");
```

Comment:
couldn't be so noisy?

Reply:
It hasn't been so far, that code is legacy